cap-std/src/fs/dir.rs

```diff
@@ -584,6 +584,19 @@ impl Dir {
         self.metadata(path).is_ok()
     }
 
+    /// Returns `true` if the path points at an existing entity.
+    ///
+    /// This corresponds to [`std::path::Path::try_exists`], but only
```

Comment:
Here and in the other instances of this, I'd prefer to reference `std::fs::try_exists` here, because most of the `Path` forms are just aliases for the `std::fs` forms, and I find it conceptually unfortunate that an otherwise I/O-free `Path` type has these I/O-performing convenience aliases attached to it.